Announcing social co-working sessions (#240)
ropensci/roweb3: content/blog/2021-08-17-coworking-sessions/index.md

output:
  html_document:
    keep_md: yes
---

We're excited to announce that as a result of our [Label-athon experiments](/events/labelathon02/), we'll be hosting monthly social coworking sessions via Zoom, starting September 2021!

Coworking is a great way to be productive and reduce feelings of social isolation (especially important over the last year)!

These sessions will give you the opportunity to 

- Meet other package developers and rOpenSci staff
- Co-work independently on work related to R (and perhaps focus on work that often gets neglected)
- Get answers to your questions

We plan to host sessions on the first Tuesday of the month, alternating times to accommodate different parts of the world.

Our first co-working session will take place **[Tuesday September 7th 2021 9-11am Pacific](/events/coworking-2021-09/)**. 

## Format

We'll be following the format we developed during our experimental label-athons: 

1. We start with introductions, giving everyone a chance to socialize and network before getting down to business
2. We co-work for 25min either in a Quiet or Noisy rooms 
    - Mix and match as you like

>     - 'Noisy' room is for chatting, socializing, discussing problems and asking questions

Reviewer: ```suggestion
    - 'Noisy' room is for discussing problems, asking questions, and chatting 
```

Reviewer: putting the work stuff before the social

3. We have 5-min breaks with fun scavenger hunts to get you up and moving

> 4. Repeat! We'll do a total of 3x 25min co-working blocks each separated by a 5-min break

Author: Right, I think I was worried that 10 min wasn't enough for intro, but I'll be cutting it down and we'll just have people introduce themselves adn move on

> 4. Repeat! We'll do a total of 3x 25min co-working blocks each separated by a 5-min break

Author: I made the change

5. Wrap up, a chance to share what we did and say good bye

## Label-athon experiments

In case you missed it, as a follow up to the community call [Set Up Your Package to Foster a Community](commcalls/apr2021-pkg-community/), 
this past spring and summer we experimented with hosting four co-working 
[label-athon events](/events/labelathon02/) on Zoom. 

During these events, participants introduced themselves and shared what they wanted to work on. 
We had two Zoom rooms, a quiet work room and a 'noisy' chat room, where we could socialize, ask questions, or discuss our work.

In line with the theme of the community call, we encouraged participants to work on making their packages more contributor friendly, such as labelling issues to invite contributions, or creating road maps. 
However, this wasn't a requirement for participation and many people worked on other things such as general package development and organization, research and planning for future development, setting up personal GitHub READMEs and preparing useR! 2021 presentations. 

> We set up a work-flow to identify rOpenSci packages with issues labelled "help wanted", so we can tweet anything that came out of the co-working events (we'd tweet non-rOpenSci packages too, but needed participants to let us know which ones they worked on). 

Reviewer: ```suggestion
We set up a work-flow to identify rOpenSci packages with issues labelled "help wanted", so we can tweet anything that came out of the co-working events. 
```

---
title: rOpenSci Introduces Monthly Social Coworking and Office Hours
author:
  - Steffi LaZerte
  - Stefanie Butland
  - Nicholas Tierney
date: '2021-08-17'
slug: coworking-sessions
categories: []
tags:
  - coworking
  - labelathon
  - events
description: Work, meet other developers, get your questions answered, and/or socialize!
tweet: rOpenSci introduces monthly social coworking sessions by @steffilazerte, @StefanieButland, and @nj_tierney!
output:
  html_document:
    keep_md: yes
---

We're excited to announce that we'll be hosting monthly social coworking + office hours sessions via Zoom, starting September 7th!

Coworking is a great way to be productive and reduce feelings of social isolation (especially important over the last year)!

These sessions will give you the opportunity to 

- Meet other package developers and rOpenSci staff
- Cowork independently on work related to R (and perhaps focus on work that often gets neglected)
- Get answers to your questions

We will host sessions on the first Tuesday of the month, alternating times to accommodate different parts of the world. 
Come for the full 2 hrs or only as long as you need!

> Our first two co-working sessions will take place **[Tuesday September 7th 2021 9-11am Pacific](/events/coworking-2021-09/)** and **[Tuesday October 5th 2021 9-11am Australian Western](/events/coworking-2021-10/)**

Reviewer: ```suggestion
Our first two coworking sessions will take place **[Tuesday September 7th 2021 9-11am Pacific](/events/coworking-2021-09/)** and **[Tuesday October 5th 2021 9-11am Australian Western](/events/coworking-2021-10/)**
```

## Format

We'll be following the format we developed during our experimental label-athons: 

1. We start with introductions, giving everyone a chance to socialize and network before getting down to business

> 2. We co-work for 30min either in a Quiet or Noisy room

Reviewer: ```suggestion
2. We work for 30min either in a Quiet or Noisy room
```

    - Mix and match as you like
    - 'Noisy' room is for discussing problems, asking questions, and chatting 
3. We have 5-min breaks with fun scavenger hunts to get you up and moving

> 4. Repeat! We'll do a total of 3x 30min co-working blocks each separated by a 5-min break

Reviewer: ```suggestion
4. Repeat! We'll do a total of 3x 30min work blocks each separated by a 5-min break
```

5. Wrap up, a chance to share what we did and say good bye

## Building on our successful Label-athon experiment

In case you missed it, as a follow up to the community call [Set Up Your Package to Foster a Community](commcalls/apr2021-pkg-community/), 

> this past spring and summer we experimented with hosting four co-working 

Reviewer: ```suggestion
this past spring and summer we experimented with hosting four coworking 
```

[label-athon events](/events/labelathon02/) on Zoom. 

During these events, participants introduced themselves and shared what they wanted to work on. 
We had two Zoom rooms, a quiet work room and a 'noisy' chat room, where we could socialize, ask questions, or discuss our work.

In line with the theme of the community call, we encouraged participants to work on making their packages more contributor friendly, such as labelling issues to invite contributions, or creating road maps. 
However, this wasn't a requirement for participation and many people worked on other things such as general package development and organization, research and planning for future development, setting up personal GitHub READMEs and preparing useR! 2021 presentations. 

> We set up a work-flow to identify rOpenSci packages with issues labelled "help wanted", so we can tweet anything that came out of the co-working events. 

Reviewer: ```suggestion
We set up a work-flow to identify rOpenSci packages with issues labelled "help wanted", so we can tweet anything that came out of the coworking events. 
```

We had quite an international turn out, with 3-8 participants per event from 12 countries. 

> Feedback was overwhelmingly positive, with participants enjoying the opportunity to work on neglected projects, ask questions about package development, meet other devs and participate in the fun scavenger hunts 😸

Reviewer: ```suggestion
Feedback was overwhelmingly positive, with participants enjoying the opportunity to work on neglected projects, ask questions about package development, meet other devs and participate in the fun scavenger hunts 😸

These were a few of their favorite things:

> “The vibe. Welcoming, friendly, and down-to-business.

> It was nice to be accountable and have friends around...

> Have some feedback from like minded technical people

> [The organizers] “...did a *great* job creating a welcoming environment and helping normalize that non-coding work, like documentation, is a valuable part of software development.

> Hearing what other people were working on. Being able to ask questions. A sense of community.

Come join us!

```

---
title: rOpenSci Introduces Monthly Social Coworking and Office Hours
author:
  - Steffi LaZerte
  - Stefanie Butland
  - Nicholas Tierney
date: '2021-08-17'
slug: coworking-sessions
categories: []
tags:
  - coworking
  - labelathon
  - events
description: Work, meet other developers, get your questions answered, and/or socialize!
tweet: rOpenSci introduces monthly social coworking sessions by @steffilazerte, @StefanieButland, and @nj_tierney!
output:
  html_document:
    keep_md: yes
---

> We're excited to announce that we'll be hosting monthly social coworking + office hours sessions via Zoom, starting September 7th!

Reviewer: ```suggestion
We're excited to announce we'll be hosting monthly social coworking + office hours sessions via Zoom, starting September 7th!
```

Coworking is a great way to be productive and reduce feelings of social isolation (especially important over the last year)!

These sessions will give you the opportunity to 

- Meet other package developers and rOpenSci staff
- Cowork independently on work related to R (and perhaps focus on work that often gets neglected)
- Get answers to your questions

> We will host sessions on the first Tuesday of the month, alternating times to accommodate different parts of the world. 

Reviewer: ```suggestion
We will host sessions on the first Tuesday of each month, alternating times to accommodate different parts of the world. 
```

Come for the full 2 hrs or only as long as you need!

> Our first two coworking sessions will take place **[Tuesday September 7th 2021 9-11am Pacific](/events/coworking-2021-09/)** and **[Tuesday October 5th 2021 9-11am Australian Western](/events/coworking-2021-10/)**

Reviewer: ```suggestion
Our first two sessions take place **[Tuesday September 7th 2021 9-11am Pacific](/events/coworking-2021-09/)** hosted by [Steffi LaZerte](/author/steffi-lazerte/) and **[Tuesday October 5th 2021 9-11am Australian Western](/events/coworking-2021-10/)** hosted by [Nick Tierney](/author/nicholas-tierney/).
```

## Format

We'll be following the format we developed during our experimental label-athons: 

1. We start with introductions, giving everyone a chance to socialize and network before getting down to business
2. We work for 30min either in a Quiet or Noisy room
    - Mix and match as you like
    - 'Noisy' room is for discussing problems, asking questions, and chatting 
3. We have 5-min breaks with fun scavenger hunts to get you up and moving
4. Repeat! We'll do a total of 3x 30min work blocks each separated by a 5-min break
5. Wrap up, a chance to share what we did and say good bye

## Building on our successful Label-athon experiment

> In case you missed it, as a follow up to the community call [Set Up Your Package to Foster a Community](commcalls/apr2021-pkg-community/), 

Reviewer: ```suggestion
In case you missed it, as a follow up to the community call [Set Up Your Package to Foster a Community](/commcalls/apr2021-pkg-community/), 
```

this past spring and summer we experimented with hosting four coworking 
[label-athon events](/events/labelathon02/) on Zoom. 

During these events, participants introduced themselves and shared what they wanted to work on. 
We had two Zoom rooms, a quiet work room and a 'noisy' chat room, where we could socialize, ask questions, or discuss our work.

In line with the theme of the community call, we encouraged participants to work on making their packages more contributor friendly, such as labelling issues to invite contributions, or creating road maps. 
However, this wasn't a requirement for participation and many people worked on other things such as general package development and organization, research and planning for future development, setting up personal GitHub READMEs and preparing useR! 2021 presentations. 

> We set up a work-flow to identify rOpenSci packages with issues labelled "help wanted", so we can tweet anything that came out of the coworking events. 

Reviewer: ```suggestion
We set up a work-flow to identify rOpenSci packages with [issues labelled "help wanted"](https://github.com/search?q=org%3Aropensci+label%3A%22help+wanted%22+state%3Aopen&type=Issues), so we can tweet anything that came out of the coworking events. 
```

We had quite an international turn out, with 3-8 participants per event from 12 countries. 
Feedback was overwhelmingly positive, with participants enjoying the opportunity to work on neglected projects, ask questions about package development, meet other devs and participate in the fun scavenger hunts 😸

These were a few of their favorite things:

> > The vibe. Welcoming, friendly, and down-to-business.

Reviewer: ```suggestion
- _"The vibe. Welcoming, friendly, and down-to-business."_
- _"It was nice to be accountable and have friends around ..."_
- _"[The organizers] “...did a *great* job creating a welcoming environment and helping normalize that non-coding work, like documentation, is a valuable part of software development."_
- _"Hearing what other people were working on. Being able to ask questions. A sense of community."_
```

We set up a work-flow to identify rOpenSci packages with [issues labelled "help wanted"](https://github.com/search?q=org%3Aropensci+label%3A%22help+wanted%22+state%3Aopen&type=Issues), so we can tweet anything that came out of the coworking events. 

We had quite an international turn out, with 3-8 participants per event from 12 countries. 
Feedback was overwhelmingly positive, with participants enjoying the opportunity to work on neglected projects, ask questions about package development, meet other devs and participate in the fun scavenger hunts 😸

These were a few of their favorite things:

- _"The vibe. Welcoming, friendly, and down-to-business."_
- _"It was nice to be accountable and have friends around ..."_
- _"[The organizers] “...did a *great* job creating a welcoming environment and helping normalize that non-coding work, like documentation, is a valuable part of software development."_
- _"Hearing what other people were working on. Being able to ask questions. A sense of community."_

> > It was nice to be accountable and have friends around...

Reviewer: ```suggestion
```

> > Have some feedback from like minded technical people

Reviewer: ```suggestion
```

> > [The organizers] “...did a *great* job creating a welcoming environment and helping normalize that non-coding work, like documentation, is a valuable part of software development.

Reviewer: ```suggestion
```

> > Hearing what other people were working on. Being able to ask questions. A sense of community.

Reviewer: ```suggestion
```
